.github/workflows/build.yml

```diff
@@ -36,7 +36,19 @@ jobs:
           NEOVIM_BIN: "C:/tools/neovim/Neovim/bin/nvim.exe"
           RUST_BACKTRACE: full
         run: |
-          cargo test -- -- -Z unstable-options --format json | cargo2junit > results.xml
+          # Tag: BROKEN_CI

```

Comment:
I've added this `Tag: BROKEN_CI`, to mark broken parts and does not repeat comments.